recipe/meta.yaml

```diff
@@ -46,6 +47,7 @@ about:
   license_file: LICENSE
   summary: Style preserving TOML library
   dev_url: https://github.com/sdispater/tomlkit
+  doc_url: https://github.com/sdispater/tomlkit/blob/master/docs/quickstart.rst
```

Comment:
Should we have `doc_src` url as well? Or, in this case maybe, instead of? It looks like the author is preparing to use a different site to host documentation.

Reply:
Or maybe a comment indicating the author's intent to make it obvious to the next person that updates the recipe.

Reply:
Thanks! I added doc_source_url and updated doc_url